CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ nav_order: 1
 
 # Changelog
 
+## Unreleased
```

Comment:
please change the placeholder to be `[MAJOR.MINOR.PATCH] - YYYY-MM-DD`

Reply:
Done.